### PR TITLE
Add showing error message if somebody will suggest tag which already exists

### PR DESCRIPTION
**Pull request by YuriyLu**

### CODE REVIEW

File: app/components/FormFields/validate.js [hunk 1]
```diff
@@ -74,10 +74,14 @@ const requiredForObjectField = value => {
   return !val || (val && !val.value) ? messages.requiredField : undefined;
 };
 
+const tagNameHasNotExist = suggestedTags => value =>
```
Comment (sergeyi23) [inline app/components/FormFields/validate.js:77]: All validators in this js file are reusable. They should have generic names that can be reused in other places.

Comment (sergeyi23) [reply app/components/FormFields/validate.js:77]: There is already `valueHasNotBeInList` that can be used. No need to create new validator.

File: app/containers/CreateTag/Form.jsx [hunk 2]
```diff
@@ -38,49 +39,64 @@ export const Form = ({
...
+      <Field
+        disabled={createTagLoading}
+        name={NAME_FIELD}
+        component={TextInputField}
+        label={translations[messages.name.id]}
+        tip={translations[messages.nameTip.id]}
+        validate={[
+          strLength2x15,
+          required,
+          valueHasNotBeInList,
```
Comment (sergeyi23) [inline app/containers/CreateTag/Form.jsx:76]: No need to add additional validator. Need to add suggested tag to the list of values for `valueHasNotBeInList`. 
See line in this file: 
```
valueHasNotBeInListValidate: (
state?.toJS()?.form?.[FORM_NAME]?.values?.[FORM_COMMUNITY]?.tags ?? []
).map(x => x.name)
```
Suggested tags need to be added to that list.

**Review by sergeyi23 — CHANGES_REQUESTED** [review]
Please see my comments below.
No need to add new validator.

File: app/containers/CreateTag/Form.jsx (hunk 2 above)
Comment (YuriyLu) [reply app/containers/CreateTag/Form.jsx]: But we can get aggested tags by asynchronous action and this value isn't exist in time of creating 'valueHasNotBeInListValidate'. Do I have to do asynchronous validator?

File: app/components/FormFields/validate.js [hunk 3]
```diff
@@ -74,14 +74,11 @@ const requiredForObjectField = value => {
   return !val || (val && !val.value) ? messages.requiredField : undefined;
 };
 
-const valueHasNotBeInList = (...args) => {
+const valueHasNotBeInList = () => (...args) => {
+  console.log('123');
```
Comment (sergeyi23) [inline app/components/FormFields/validate.js:78]: Remove console.log

File: app/components/FormFields/validate.js [hunk 4]
```diff
@@ -74,14 +74,11 @@ const requiredForObjectField = value => {
...
   const value = args[0];
   const list = args[2].valueHasNotBeInListValidate;
-
-  return list &&
-    list.find(
-      x => x && x.trim().toLowerCase() === value && value.trim().toLowerCase(),
-    )
+  return list.includes(value.toLowerCase())
```
Comment (sergeyi23) [inline app/components/FormFields/validate.js:81]: Maybe `list && list.includes(value.toLowerCase())` to be safe?

**Review by sergeyi23 — CHANGES_REQUESTED** [review]
See couple of new comments

**Review by sergeyi23 — APPROVED** [review]

**Pull request merged**